<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-8SE8GSP" ignored="0" name="Default test" tests="6" failures="2" timestamp="29 Apr 2020 17:48:32 GMT" time="3.381" errors="0">
  <testcase name="loading" time="0.076" classname="com.testng.practice.TimeoutAndRetryAnalyzer">
    <skipped/>
  </testcase> <!-- loading -->
  <testcase name="loading" time="0.009" classname="com.testng.practice.TimeoutAndRetryAnalyzer">
    <skipped/>
  </testcase> <!-- loading -->
  <testcase name="loading" time="0.019" classname="com.testng.practice.TimeoutAndRetryAnalyzer">
    <skipped/>
  </testcase> <!-- loading -->
  <testcase name="loading" time="0.011" classname="com.testng.practice.TimeoutAndRetryAnalyzer">
    <skipped/>
  </testcase> <!-- loading -->
  <testcase name="loading" time="0.021" classname="com.testng.practice.TimeoutAndRetryAnalyzer">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:568)
at org.testng.Assert.assertEquals(Assert.java:578)
at com.testng.practice.TimeoutAndRetryAnalyzer.loading(TimeoutAndRetryAnalyzer.java:32)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.retryFailed(Invoker.java:839)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1010)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- loading -->
  <testcase name="pageLoading" time="3.036" classname="com.testng.practice.TimeoutAndRetryAnalyzer">
    <failure type="org.testng.internal.thread.ThreadTimeoutException" message="Method com.testng.practice.TimeoutAndRetryAnalyzer.pageLoading() didn&amp;apos;t finish within the time-out 3000">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method com.testng.practice.TimeoutAndRetryAnalyzer.pageLoading() didn't finish within the time-out 3000
at java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.signalAll(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.tryTerminate(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.processWorkerExit(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- pageLoading -->
</testsuite> <!-- Default test -->
